src/Core/src/Handlers/Button/ButtonHandler.iOS.cs

```diff
@@ -12,6 +12,9 @@ public partial class ButtonHandler : ViewHandler<IButton, UIButton>
 		static UIColor? ButtonTextColorDefaultHighlighted;
 		static UIColor? ButtonTextColorDefaultNormal;
 
+		// This appears to be the padding that Xcode has when "Default" content insets are used
+		public readonly static Thickness DefaultPadding = new Thickness(12, 7);
```

Comment:
Although buttons in iOS don't actually have a padding because for some reason iOS is very different. The new styles that come with iOS 15 and using the brand new `UIButton.Configuration` we _finally_ have a way to style buttons properly. However, since we support pre-15, we still use the old `ContentEdgeInsets` property and I just replicated those values. For a perfect "native" feel, just set the padding to 0 (and wonder why I am crying).

In this case, I measured the difference between "Default" and a "Custom" 0 padding, and this is what I got.